server/hedley/modules/custom/hedley_migrate/csv/geolocations_burundi.csv

```diff
@@ -0,0 +1,80 @@
+Bururi,Vyanda,Gitsiro,Mushishi,Gitega

```

Comment:
Do we need to execute this migration on `live` in the end? In case, will it be a manual step?

Reply:
`geolocations_[Site].csv` files are used to generate GeoFields dictionaries for front-end.
So, there are no migrations for these files